src/include/mkdirp/mkdirp.h

```diff
@@ -0,0 +1,21 @@
+
+//
+// mkdirp.h
+//
+// Copyright (c) 2013 Stephen Mathieson
+// MIT licensed
```

Comment:
We should have a full copy of the MIT license from the mkdirp library in here, since we are copying and redistributing that code.